### ISSUE TITLE
Event log

### ISSUE THREAD

**Issue by mwhamgenomics** [EdinburghGenomics/Reporting-App#35]
We are already storing the starts and ends of pipeline stages, but we should expand on this.

For samples, run elements, etc., we should store review/comment dates. For this, we should have an `events` endpoint. Suggested schema:

``` yaml
events:
    event_id: {type: 'string', unique: true}  # e.g. run_review_2016-04-14_12:00:00
    event_type: {type: 'string'}  # e.g. 'stage start', 'run_review'
    payload: {type: 'string'}  # e.g. 'pass'
```

**Comment by tcezard** [comment]
if we're going to create events then we should also store the user performing the event.
We should also investigate how these events can be store on the LIMS and retrieved using #29

**Comment by tcezard** [comment]
This was implemented #123. 
Closing.
